input/pages/smart-dynreg.md

```diff
@@ -0,0 +1,119 @@
+This page outlines a protocol for [public applications](app-launch.html#support-for-public-and-confidential-apps) to request persistent API authorization on behalf of a user or patient. It differs from the [UDAP specification](https://www.udap.org/) in that it allows a purely public app (one without any static credential) to request persistant API access.
+
+### Background
+With the existing public app workflow, the app is not given a refresh token or any other way to request new access tokens. This means the app is limited to using APIs for the time its first access token is valid.
```

Comment:
The existing public app workflow supports public apps with persistent access using refresh tokens, doesn't it?